src/graph/graph.c

```diff
@@ -1029,6 +1048,10 @@ static void _BulkDeleteNodes(Graph *g, Node *nodes, uint node_count,
 		 * A will contain all implicitly deleted edges from R */
 		GrB_Matrix_apply(A, Mask, GrB_NULL, GrB_IDENTITY_UINT64, R, desc);
 
+		// Decrement number of edges by the number of setted bits in A
+		GrB_Matrix_nvals(&nvals, A);
```

Comment:
in the case of multi-edge `nvals` is not the way to determine number of edges deleted,
use:
1. x = Datablock item count prior to deletion
2. y = Datablock item count after to deletion
3. removed = x - y (total number of edges of type `r` removed by this deletion)